### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by jtbx**
Fixes conflicts in #57

### CODE REVIEW

File: src/engine/docs/snapshots.txt [hunk 1]
```diff
@@ -4,12 +4,12 @@ Section: Overview
...
 Topic: Definitions
 
-- *Snapshot*. A is a serialized game state from which a client can render the game from. They are sent from the server at a regular interval and is created specifically for each client in order to reduce bandwidth.
+- *Snapshot*. A serialized game state from which a client can render the game from. They are sent from the server at a regular interval and is created specifically for each client in order to reduce bandwidth.
 - *Delta Snapshot*. A set of data that can be applied to a snapshot in order to create a new snapshot with the updated game state.
 
 Topic: Structure
 
-A snapshot contains a series of items. Each item have a type, id and data.
+A snapshot contains a series of items. Each item has a type, an id and data.
```
Comment (ChillerDragon) [inline src/engine/docs/snapshots.txt:12]: I am not a native but "type,  id and data." reads smoother to me. Do you put a and an in comma separated lists?

**Review by AssassinTee — APPROVED** [review]
LGTM

**Review by MilkeeyCat — APPROVED** [review]

**Review by ST-Chara — APPROVED** [review]

File: src/engine/docs/snapshots.txt (hunk 1 above)
Comment (ChillerDragon) [reply src/engine/docs/snapshots.txt:12]: Yea it also says "a type" and nobody else seems to be confused by this. So LGTM.

**Review by ChillerDragon — APPROVED** [review]

File: src/engine/docs/snapshots.txt (hunk 1 above)
Comment (AssassinTee) [reply src/engine/docs/snapshots.txt:12]: @ChillerDragon it has a single type field, a single id field, and data, which doesn't have a singular or plural

Comment (ChillerDragon) [reply src/engine/docs/snapshots.txt:12]: Okay thanks :england: :+1:

**Pull request merged**